resolwe_bio/processes/workflows/gene-fusion-calling-arriba.py

```diff
@@ -0,0 +1,256 @@
+                description="Maximum number of loci the read is allowed to map to. Alignments "
+                "(all of them) will be output only if the read maps to no more loci than this "
+                "value. Otherwise no alignments will be output, and the read will be counted as "
+                "'mapped to too many loci' (default: 10).",
+            )
+
+        star_chimeric = GroupField(
+            ChimericReadsOptions,
+            label="Alignment with STAR",
+        )
+
+    class Output:
+        """Output fields."""
+
+    def run(self, inputs, outputs):
+        """Run the workflow."""
+
+        if inputs.reads and inputs.bam:
+            self.error("Select either FASTQ or BAM file, not both.")
+
+        if not inputs.reads and not inputs.bam:
+            self.error("You must provide either FASTQ or BAM input.")
+
+        if inputs.bam:
+            bam = inputs.bam
+
+        elif inputs.reads:
+            input_star = {
+                "reads": inputs.reads,
+                "genome": inputs.star_index,
+                "gene_counts": True,
+                "detect_chimeric": {
+                    "chimeric": inputs.star_chimeric.chimeric,
+                    "chim_segment_min": inputs.star_chimeric.chim_segment_min,
+                    "chim_out_type": inputs.star_chimeric.chim_out_type,
+                    "chim_junction_overhang_min": inputs.star_chimeric.chim_junction_overhang_min,
+                    "chim_score_drop_max": inputs.star_chimeric.chim_score_drop_max,
+                    "chim_score_junction_non_gtag": inputs.star_chimeric.chim_score_junction_non_gtag,
+                    "chim_score_separation": inputs.star_chimeric.chim_score_separation,
+                    "chim_segment_read_gap_max": inputs.star_chimeric.chim_segment_read_gap_max,
+                    "chim_multimap_nmax": inputs.star_chimeric.chim_multimap_nmax,
+                    "pe_overlap_n_bases_min": inputs.star_chimeric.pe_overlap_n_bases_min,
+                    "align_spliced_mate_map_lmin_over_lmate": inputs.star_chimeric.align_spliced_mate_map_lmin_over_lmate,
+                    "align_sj_stitch_mismatch_n_max": inputs.star_chimeric.align_sj_stitch_mismatch_n_max,
+                },
+                "filtering": {
+                    "out_multimap_max": inputs.star_chimeric.out_multimap_max,
+                },
+            }
+
+            bam = Data.create(
+                process=BioProcess.get_latest(slug="alignment-star"),
+                input=input_star,
+                name=f"Aligned ({inputs.reads.name})",
+            )
+
+        arriba_input = {
+            "bam": bam.id,
+            "gtf": inputs.gtf.id,
+            "genome": inputs.genome.id,
+        }
+
+        if inputs.blacklist_file:
+            arriba_input["blacklist_file"] = inputs.blacklist_file.id
+
+        fusion_name = (
+            f"Fusions ({inputs.bam.name})" if inputs.bam else "Fusions (from reads)"
```

Comment:
Why not f"Fusions ({inputs.bam.name})" if inputs.bam else f"Fusions ({inputs.reads.name})"?